autoload/airline/extensions/branch.vim

```diff
@@ -206,13 +251,15 @@ function! s:check_in_path()
 endfunction
 
 function! s:reset_untracked_cache(shellcmdpost)
```

Comment:
Instead of in-function comment, consider documenting 'shellcmdpost' argument in a function-level comment:
`" shellcmdpost - whether function was called as a result of ShellCmdPost hook`

This is the interface of the function, therefore an outer comment suits it.